aws/logs_monitoring/template.yaml

```diff
@@ -433,6 +433,14 @@ Resources:
               Action:
                 - s3:GetObject
               Resource: "*"
+            - Effect: Allow
+              Action:
+                - kms:Encrypt
+                - kms:Decrypt
```

Comment:
@guitarrapc According to my tests, if you use the AWS managed CMK (i.e., key `aws/s3`), no extra permissions are required for the Forwarder to download the object. If you use a customer managed CMK, the Forwarder only needs `kms:Decrypt` for downloading. I guess the other kms permissions you listed are required for uploading objects to the bucket, which means they are not needed by the Forwarder? Can you confirm? Thanks!

Reply:
That make sence. AS CloudTrail only allow CMK to encypt with SSE, I foced use my CMK.
Let me check permission at my lab.

Reply:
@tianchu Thanks, I've tested on my lab and confirm only `kms:Decrypt` is needed for downloading.
I've rebase and pushed fix.
can you review?